sktime/forecasting/sarimax.py

```diff
@@ -0,0 +1,180 @@
+# -*- coding: utf-8 -*-
+# !/usr/bin/env python3 -u
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implements SARIMAX."""
+
+__all__ = ["SARIMAX"]
+__author__ = ["TNTran92"]
+
+from statsmodels.tsa.api import SARIMAX as _SARIMAX
+
+from sktime.forecasting.base.adapters import _StatsModelsAdapter
+
+
+class SARIMAX(_StatsModelsAdapter):
+    """SARIMAX forecaster.
+
+    Direct interface for `statsmodels.tsa.api.SARIMAX`.
+
+    Parameters
+    ----------
+    endog : array_like
+        The observed time-series process :math:`y`
+    exog : array_like, optional, defaul=None
+        Array of exogenous regressors, shaped nobs x k.
```

Comment:
pls remove thos, this is what we have via `fit` with `y` and `X`

Reply:
Will do. It'll be included in the commit